src/aliases/api/handlers/handlers.go

```diff
@@ -0,0 +1,297 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"sync"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	jsonutils "github.com/consensys/quorum-key-manager/pkg/json"
+	"github.com/consensys/quorum-key-manager/src/aliases/api/types"
+	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+	infrahttp "github.com/consensys/quorum-key-manager/src/infra/http"
+	"github.com/gorilla/mux"
+)
+
+type AliasHandler struct {
+	alias aliasent.AliasBackend
+}
+
+func NewAliasHandler(backend aliasent.AliasBackend) *AliasHandler {
+	h := AliasHandler{
+		alias: backend,
+	}
+
+	return &h
+}
+
+func (h *AliasHandler) Register(r *mux.Router) {
+	regRoute := r.PathPrefix("/registries/{registry_name}").Subrouter()
+	regRoute.HandleFunc("", h.deleteRegistry).Methods(http.MethodDelete)
+
+	alRoute := regRoute.PathPrefix("/aliases").Subrouter()
+	alRoute.HandleFunc("", h.listAliases).Methods(http.MethodGet)
+	alRoute.HandleFunc("/{alias_key}", h.createAlias).Methods(http.MethodPost)
+	alRoute.HandleFunc("/{alias_key}", h.getAlias).Methods(http.MethodGet)
+	alRoute.HandleFunc("/{alias_key}", h.updateAlias).Methods(http.MethodPut)
+	alRoute.HandleFunc("/{alias_key}", h.deleteAlias).Methods(http.MethodDelete)
+}
+
+// @Summary Delete a registry
+// @Description Delete a registry and all its keys
+// @Tags registry
+// @Param registry_name path string true "registry identifier"
+// @Success 204 {object} types.DeleteRegistryResponse "Registry data"
```

Comment:
Struct `DeleteRegistryResponse` does not exist. Please, make sure you can run `$> make gen-swagger` using this annotation, because I believe it will not work

Reply:
It stops at:
```
2021/09/07 09:28:06 Generate swagger docs....
2021/09/07 09:28:06 Generate general API Info, search dir:./src
2021/09/07 09:28:06 Generating types.CreateKeyRequest
2021/09/07 09:28:06 Generating types.KeyResponse
2021/09/07 09:28:06 Generating entities.Annotation
2021/09/07 09:28:06 Generating handlers.ErrorResponse
2021/09/07 09:28:06 Generating types.ImportKeyRequest
2021/09/07 09:28:06 Generating types.SignBase64PayloadRequest
2021/09/07 09:28:06 Generating types.CreateEth1AccountRequest
2021/09/07 09:28:06 Generating types.Eth1AccountResponse
2021/09/07 09:28:06 Generating types.ImportEth1AccountRequest
2021/09/07 09:28:06 Generating types.UpdateEth1AccountRequest
2021/09/07 09:28:06 Generating types.SignMessageRequest
2021/09/07 09:28:06 Generating types.SignTypedDataRequest
2021/09/07 09:28:06 Generating types.DomainSeparator
2021/09/07 09:28:06 Generating types.Type
2021/09/07 09:28:06 Generating types.SignETHTransactionRequest
2021/09/07 09:28:06 Generating types.SignEEATransactionRequest
2021/09/07 09:28:06 Generating types.SignQuorumPrivateTransactionRequest
2021/09/07 09:28:06 Generating types.ECRecoverRequest
2021/09/07 09:28:06 ParseComment error in file src/stores/api/handlers/eth1.go :cannot find type definition: types.VerifyEth1SignatureRequest
make: *** [Makefile:129: gen-swagger] Error 
```

Reply:
this was fixed on my last PR. Please rebase main I would expect following errors to be linked to your Aliases API docs

Reply:
done